content/_data/changelogs/lts.yml

```diff
@@ -2962,6 +2962,46 @@
       authors:
         - jeffret-b
 
+- version: 2.204.2
+  date: 2019-01-29
+  changes:
+    - type: major bug
+      pull: 4394
+      issue: 59107
+      authors:
+        - jvz
+      message: |-
+        User is no longer logged out when authenticating another user.
```

Comment:
This entry would benefit from clear explanation of the impact on users.

Reply:
> This entry would benefit from clear explanation of the impact on users.

@jvz can you suggest phrasing that would make it clearer to users when this happens and how it will impact the user?

The bug report says:

> As user "A" when configuring authorization using the "Run as Specific User" strategy to run a job as user "B" after successful authentication with the password of user "B" user "A" is logged out.

When the report says 'user "A" is logged out', I assume that means that user "A" is no longer authenticated in their web browser running that session.  Have I understood correctly, or is there something more subtle happening when 'user "A" is logged out'?

Reply:
Their session is cleared, yes.